### PR TITLE
Feature/vscode http integration

### PR DESCRIPTION
**Pull request by colombod**

### CODE REVIEW

File: src/dotnet-interactive-vscode/src/interactiveClient.ts [hunk 1]
```diff
@@ -38,11 +38,19 @@ import { editorLanguagesToKernelNames } from './interactiveNotebook';
...
+    private deferredCommandEventListener: KernelEventEnvelopeObserver;
 
     constructor(readonly kernelTransport: KernelTransport) {
         kernelTransport.subscribeToKernelEvents(eventEnvelope => this.eventListener(eventEnvelope));
+        this.deferredCommandEventListener = (_) =>{
+
+        };
     }
 
+    public setDeferredCommadnEventsListener(listener: KernelEventEnvelopeObserver){
```
Comment (brettfo) [inline src/dotnet-interactive-vscode/src/interactiveClient.ts:50]: `...Commadn...` => `...Command...`

File: src/dotnet-interactive-vscode/src/interactiveClient.ts [hunk 2]
```diff
@@ -209,7 +217,7 @@ export class InteractiveClient {
             });
             await this.kernelTransport.submitCommand(command, commandType, token);
         });
-    }
+    }
```
Comment (brettfo) [inline src/dotnet-interactive-vscode/src/interactiveClient.ts:220]: This whitespace kills me.

File: src/dotnet-interactive-vscode/src/interactiveClient.ts [hunk 3]
```diff
@@ -243,6 +251,9 @@ export class InteractiveClient {
                     listener(eventEnvelope);
                 }
             }
+            else{
```
Comment (brettfo) [inline src/dotnet-interactive-vscode/src/interactiveClient.ts:254]: `} else {` on a single line.

File: src/dotnet-interactive-vscode/src/interactiveClient.ts (hunk 1 above)
Comment (colombod) [reply src/dotnet-interactive-vscode/src/interactiveClient.ts:50]: Done

File: src/dotnet-interactive-vscode/src/interactiveClient.ts (hunk 2 above)
Comment (colombod) [reply src/dotnet-interactive-vscode/src/interactiveClient.ts:220]: done

File: src/dotnet-interactive-vscode/src/interactiveClient.ts (hunk 3 above)
Comment (colombod) [reply src/dotnet-interactive-vscode/src/interactiveClient.ts:254]: Done

**Pull request merged**